database.py

```diff
@@ -21,6 +21,12 @@ def getData(team_number):
     matchlist = list(dict(col)['matches'])
     return matchlist
 
+def getData(team_number):
```

Comment:
This function should be named something like getPitScoutingData so that it does not colide with the getData function.